problem_builder/step.py

```diff
@@ -52,6 +52,12 @@ def steps(self):
         """
         return [_normalize_id(child_id) for child_id in self.children if child_isinstance(self, child_id, StepMixin)]
 
+    def get_steps(self):
```

Comment:
@e-kolpakov Using `lazy` would be nice, but it would force this to become an attribute rather than a method, and I'd like this to be a method so that `block.steps` and `block.get_steps()` follows the same API pattern as `block.children / block.get_children()` and `block.parent / block.get_parent()`